z003-drivers.md

```diff
@@ -6,53 +6,56 @@ permalink: /drivers/
 
-And to ease implementation, a test interface, a veritable TCK, is provided to ensure you are fully operational.
+To ease implementation, R2DBC also provides a test interface (a veritable TCK) to ensure your driver is fully operational.
 
 # Existing Drivers
 
-R2DBC drivers implements the SPI listed above. The ones currently supported include:
+R2DBC drivers implement the SPI listed above. The ones currently supported include:
 
-* [cloud-spanner-r2dbc](https://github.com/GoogleCloudPlatform/cloud-spanner-r2dbc) - Driver for Google Cloud Spanner.
-* [jasync-sql](https://github.com/jasync-sql/jasync-sql) - R2DBC wrapper for Java & Kotlin Async Database Driver for MySQL and PostgreSQL written in Kotlin.
-* [r2dbc-h2](https://github.com/r2dbc/r2dbc-h2) - Native driver implemented for H2 as a test database.
-* [r2dbc-postgres](https://github.com/r2dbc/r2dbc-postgresql) - Native driver implemented for PostgreSQL.
-* [r2dbc-mssql](https://github.com/r2dbc/r2dbc-mssql) - Native driver implemented for Microsoft SQL Server.
-* [r2dbc-mysql](https://github.com/mirromutth/r2dbc-mysql) - Native driver implemented for MySQL.
+* [cloud-spanner-r2dbc](https://github.com/GoogleCloudPlatform/cloud-spanner-r2dbc) - driver for Google Cloud Spanner.
+* [jasync-sql](https://github.com/jasync-sql/jasync-sql) - R2DBC wrapper for Java & Kotlin Async Database Driver for MySQL and PostgreSQL (written in Kotlin).
+* [r2dbc-h2](https://github.com/r2dbc/r2dbc-h2) - native driver implemented for H2 as a test database.
+* [r2dbc-postgres](https://github.com/r2dbc/r2dbc-postgresql) - native driver implemented for PostgreSQL.
+* [r2dbc-mssql](https://github.com/r2dbc/r2dbc-mssql) - native driver implemented for Microsoft SQL Server.
+* [r2dbc-mysql](https://github.com/mirromutth/r2dbc-mysql) - native driver implemented for MySQL.
 
 # Connection Pooling
 
 The following connection pool implementations are available for R2DBC:
 
-* [r2dbc-pool](https://github.com/r2dbc/r2dbc-pool) - Reactive connection pooling using Reactor Pool.
+* [r2dbc-pool](https://github.com/r2dbc/r2dbc-pool) - reactive connection pooling using Reactor Pool.
 
-# Implementing a driver
+# Implementing a Driver
 
-R2DBC has a clearly defined **SPI** you must implement to host a solution for your data store. To build an implementation, add the following artifact to your build:
+R2DBC has a clearly-defined SPI, which you must implement to host a solution for your data store. To build an implementation, add the following artifact to your build:
 
-* Group: **io.r2dbc**
-* Artifact: **r2dbc-spi**
+* <strong>Group</strong> <code>io.r2dbc</code>
+* <strong>Artifact</strong> <code>r2dbc-spi</code>
 
-The key interface all driver providers must implement is the [`Connection`](https://r2dbc.io/spec/0.8.0.RELEASE/api/io/r2dbc/spi/Connection.html) and a set of other interfaces.
+* **Group** `io.r2dbc`
+* **Artifact** `r2dbc-spi`
+
```

Comment:
May wish to be consistent on apply strong and code formatting.

Reply:
Yeah, I think I just used this to describe the artifact. I might instead pull this out from being a list and put the artifact and group inline.

Reply:
Ah, I see what happened… I had left one version with HTML tags and another with Markdown, so there were two listings for the same thing. Oops. Fixed! :)